packages/browser-wallet/src/popup/shared/Form/common/utils.tsx

```diff
@@ -121,7 +121,7 @@ export const makeUncontrolled = <E, P extends RequiredUncontrolledFieldProps<E>>
         // Filter away all props required for form registration, leaving the props for the input '<Field />'
         const { name, rules, register, onChange: ownOnChange = noOp, onBlur: ownOnBlur = noOp, ...ownProps } = props;
         const registerProps = register(name, rules);
-        const { errors, touchedFields, isSubmitted } = useFormState<TFieldValues>(name);
+        const { errors, touchedFields, isSubmitted } = useFormState<TFieldValues>({ name });
```

Comment:
:thinking:

Reply:
I don't know.. It complained after upgrading typescript, and looking at the function signature and the documentation, I don't know why it was accepted prior to this..

https://react-hook-form.com/docs/useformstate